core/src/main/scala/tech/beshu/ror/accesscontrol/ops.scala

```diff
@@ -152,7 +153,6 @@ object show {
           showTraversable("indices", bc.indices) ::
           showOption("kibana_idx", bc.userMetadata.kibanaIndex) ::
           showTraversable("response_hdr", bc.responseHeaders) ::
-          showTraversable("context_hdr", bc.contextHeaders) ::
```

Comment:
we don't want to show context headers any more (because there is no such things any more), but this log is helpful for support to figure out what was resolved in the matched block - it seems that we need to place here all information which will be helpful to figure out what was collected by the fields rule.